requirements-dev.txt

```diff
@@ -12,6 +12,7 @@ mkdocs-material>=7
 mkdocstrings>=0.15
 mypy
 pandas
+pandas-path>=0.3.0
```

Comment:
Wonder if it makes sense to add a `pandas` extras option.

Reply:
I think it's kind of conceptually nice to have the extras be focused on cloud providers since that's the major differentiator.